README.md

```diff
@@ -5,10 +5,14 @@ This is the codebase for the NERD Summit website: https://nerdsummit.org/
 
 The site is essentially a one-page website with most of the content in `index.html` and with `code.html` being the Code of Conduct page. 
 
-No SASS is used, just CSS.
-
 The sessions section is built with Javascript via the `/js/sessions.js` file which gets the sessions data from `data/sessions.json`. This is currently in the process of being changed to get data from a Google Sheet via a JSON feed.  See BACKEND section.
 
+# FRONTEND
+
+Starting with the 2020 version of the site, Gulp has been configured to use Sass and PostCSS. In order to use this, install everything by opening a terminal window, navigating to the `nerdsummit2020` folder on your machine, and running `npm install`. To run the Gulp command, run `gulp watch`.
+
+CSS classes should be named by using the BEM (Block, Element, Modifier) methodology. To learn more, [Get BEM](http://getbem.com/) give an overview of the methodology, ways to implement it, and answers frequently asked questions.
+
```

Comment:
Added this paragraph about using BEM. Let me know if I need to add any more information about it!